neuro_extras/main.py

```diff
@@ -452,13 +403,20 @@ async def _nonstorage_cp(source_url: URL, destination_url: URL) -> None:
         if source_path.is_dir():
             dir_util.remove_tree(str(source_path))
         else:
-            source_path.unlink()
+            if source_path.exists():
+                source_path.unlink()
+
+
+@main.group()
+def data() -> None:
+    pass
 
 
 @data.command("cp")
 @click.argument("source")
 @click.argument("destination")
 @click.option("-x", "--extract", default=False, is_flag=True)
+@click.option("-c", "--compress", default=False, is_flag=True)
```

Comment:
Could we add some hints about the code behaviour? (help for `-x` and `-c` parameter).
For instance:
 "--extract" `Perform extraction of SOURCE into the temporal folder and move extracted files to DESTINATION. The archive type is derived from the file name. Supported types: .tar.gz, .tgz, .tar.bz2, .tbz2, .tbz, .tar, .gz, .zip.`
and for 
 "--compress" `Perform compression of SOURCE into the temporal folder and move created archive to DESTINATION. The archive type is derived from the file name. Supported types: .tar.gz, .tgz, .tar.bz2, .tbz2, .tbz, .tar, .gz, .zip.`

Also. It may be beneficial to keep a single list of supported archive types and perform input validation before downloading BLOB and trying to extract it. We may also print this list [here](https://github.com/neuro-inc/neuro-extras/blob/an/38-packing/neuro_extras/main.py#L345).